doc/articles/get-started-vscode.md

```diff
@@ -74,8 +74,6 @@ You may need to follow additional directions, depending on your development envi
 
 ## Next Steps
 
-You're all set! Now that you're done with setting up your environment.
-
-Learn more about:
+You're all set! Now that you're done with setting up your environment, learn more about:
```

Comment:
Changes to the get started are not needed, they've been aligned in a separate PR.

Reply:
@jeromelaban 
In this case, can this PR be closed?
Wouldn’t standardizing ## Next Steps with text and topic be a good option?